edi_endpoint_oca/models/edi_endpoint.py

```diff
@@ -18,7 +13,7 @@ class EDIEndpoint(models.Model):
     """
 
     _name = "edi.endpoint"
-    _inherit = "endpoint.mixin"
+    _inherit = ["endpoint.mixin"]
```

Comment:
Your test issue was coming from there? Should it deserves a comment explaining this, someone that will migrate this later could remove these square brackets.

Reply:
no no, this is a left over / non necessary change. I added  the sync mixin here 1st then I removed it ;)